### PR TITLE
[fix] remove import and fix typescript error

### PR DESCRIPTION
**Pull request by vuvu2259**

### CODE REVIEW

File: core/ui/src/declarations.d.ts [hunk 1]
```diff
@@ -0,0 +1,5 @@
+declare namespace JSX {
+  interface IntrinsicElements {
+    'model-viewer': any;
+  }
+}
```
Comment (haVincy) [inline core/ui/src/declarations.d.ts:5]: I think we don't need a new file, we already have `.d.ts` under `core/ui/src`?

File: core/ui/src/public/OrderDetail/index.tsx [hunk 2]
```diff
@@ -112,7 +111,13 @@ export const OrderDetail: React.FC<OrderDetailProps> = ({
...
     <div className="candy-order-detail">
       <div className="candy-container">
-        <div className="candy-order-detail-left">{order && <Viewer order={order} />}</div>
+        <div className="candy-order-detail-left">
+          {order && order.nftAnimationLink ? (
+            <video width="100%" height="auto" controls autoPlay loop src={order.nftAnimationLink}></video>
+          ) : (
+            <LiqImage src={order?.nftImageLink || ''} alt={order?.name} fit="contain" />
+          )}
+        </div>
```
Comment (haVincy) [inline core/ui/src/public/OrderDetail/index.tsx:120]: Why we have to change this if we can fix the error by declaration?

File: core/ui/src/declarations.d.ts (hunk 1 above)
Comment (vuvu2259) [reply core/ui/src/declarations.d.ts:5]: I tried to add in `.d.ts` but it isn't work. You can try!

File: core/ui/src/public/OrderDetail/index.tsx (hunk 2 above)
Comment (vuvu2259) [reply core/ui/src/public/OrderDetail/index.tsx:120]: @beepsaidblue said the order detail don't need to show 3d model so I change it in this branch as well
This isn't fix for the typescript error

File: core/ui/src/declarations.d.ts (hunk 1 above)
Comment (haVincy) [reply core/ui/src/declarations.d.ts:5]: I tried but works for me?

```ts
declare namespace JSX {
  interface IntrinsicElements {
    "model-viewer": ModelViewerElement;
  }
}
```

**Review by haVincy — APPROVED** [review]

File: core/ui/src/public/OrderDetail/index.tsx (hunk 2 above)
Comment (beepsaidblue) [reply core/ui/src/public/OrderDetail/index.tsx:120]: i meant OrderDetail don't need "import '@google/model-viewer". we should keep the Viewer here

**Review by beepsaidblue — APPROVED** [review]

**Pull request merged**